### ISSUE TITLE
Unable to load data

### ISSUE THREAD

**Issue by EndNation** [Cvolton/GMDprivateServer#493]
So apparently, the private server decided to not be able to load data. Everything else is working fine, and my initial data before this is still there. I went to check the keys in data/accounts/keys/${accountId} and the file is empty. Should it be empty or is something wrong?

**Comment by Cvolton** [comment]
key files can be empty if you didn't enable save data encryption (which is something I advise against nowadays because it breaks password resets)

**Comment by Cvolton** [comment]
> Check if you can see the data through the weblink, e.g. **<yourgdpsURL>/<database>/data/incl/<filename>**
> If you get 403, you didn't set up the permissions for the folder properply.
> If you get an Internal Server Error, it's most likely that the file can't be shown due to the size. If you host your GDPS on a VPS, you can configurate the limit in the webserver configs. If you host your GDPS on a free webhosting service (such as 7m.pl), you may will have issues and need to switch to a different server unless you have the ability to change the limit there too.

this is wrong, _<yourgdpsURL>/<database>/data/incl/<filename>_ should never work on a properly set up GDPS (not to mention that the directory is wrong and it's actually _/data/accounts/<filename>_), outside access to that directory is disabled in .htaccess due to security concerns. Error 403 is the intended behaviour here
